plugins/org.jboss.tools.openshift.ui/plugin.xml

```diff
@@ -756,21 +756,29 @@
 			class="org.jboss.tools.openshift.internal.ui.handler.DeleteResourceHandler"
 			commandId="org.eclipse.ui.edit.delete">
 				<activeWhen>
-					<or>
+					<and>
 						<count
 							value="1">
 						</count>
 						<iterate>
 							<and>
+							  <or>
 								<instanceof
 									value="com.openshift.restclient.model.IResource" />
-								<not>
+      							<instanceof
```

Comment:
I think it should adapt to a IResource so deletion works from the Properties view